themebuilder/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+    "env": {
+        "node": true,
+        "mocha": true
```

Comment:
Isn't it rule for `tests` folder only?

Reply:
We need 'node: true' rule for 'cli' folder. We use global 'process' variable to retrieve command line arguments. Do you think it is better to split '.eslinrc' for 'tests' and 'cli'?

Reply:
I prefer to minimize the scope of the rule as it's possible to avoid any unexpected usage of `mocha` globals.